tests/test_core.py

```diff
@@ -884,7 +884,7 @@ def test_sup(self):
       self.do_run(src, 'sizeofs:6,8\n*C___: 0,6,12,20<24*\n*Carr: 0,6,12,20<24*\n*C__w: 0,6,12,20<24*\n*Cp1_: 4,6,12,20<24*\n*Cp2_: 0,6,12,20<24*\n*Cint: 0,8,12,20<24*\n*C4__: 0,8,12,20<24*\n*C4_2: 0,6,10,16<20*\n*C__z: 0,8,16,24<28*')
 
   def test_assert(self):
-    self.do_run_in_out_file_test('tests', 'core', 'test_assert')
+    self.do_run_in_out_file_test('tests', 'core', 'test_assert', assert_returncode=None)
```

Comment:
Should we be checking for nonzero in cases like this?  Do we need yet another arg for "expect fail"?

Reply:
Perhaps that would be even better, yes. I don't like adding yet more arguments, but it would be a further improvement on this PR. Do you think it's worth it?